store/storetest/user_store.go

```diff
@@ -2310,12 +2327,15 @@ func testUserUnreadCount(t *testing.T, ss store.Store) {
 	p1 := model.Post{}
 	p1.ChannelId = c1.Id
 	p1.UserId = u1.Id
-	p1.Message = "this is a message for @" + u2.Username
+	p1.Message = "this is a message for @" + u2.Username + " and " + "@" + u3.Username
 
 	// Post one message with mention to open channel
 	_, nErr = ss.Post().Save(&p1)
 	require.NoError(t, nErr)
-	nErr = ss.Channel().IncrementMentionCount(c1.Id, u2.Id, false)
+	nErr = ss.Channel().IncrementMentionCount(c1.Id, []string{u2.Id}, false)
+	require.NoError(t, nErr)
+
+	nErr = ss.Channel().IncrementMentionCount(c1.Id, []string{u3.Id}, false)
```

Comment:
Wouldn't the actual scenario be to increment both at the same time?

```suggestion
	nErr = ss.Channel().IncrementMentionCount(c1.Id, []string{u2.Id, u3.Id}, false)
```

Reply:
Oh, brain.exe stopped working for a while there.